### ISSUE TITLE
CmdDel fails if NAD contains chained CNI plugins

### ISSUE THREAD

**Issue by TothFerenc** [k8snetworkplumbingwg/whereabouts#102]
Example NAD which contains a chained `sbr` CNI:
```
apiVersion: "k8s.cni.cncf.io/v1"
kind: NetworkAttachmentDefinition
metadata:
 name: ipvlan-pf3-sbr
 namespace: kube-system
spec:
 config: '{
  "plugins": [
   {
    "cniVersion": "0.3.1",
    "type": "ipvlan",
    "master": "pf3",
    "mode": "l2",
    "ipam": {
     "type": "whereabouts",
     "range": "10.10.10.50-10.10.10.60/24",
     "routes": [ { "dst": "10.50.0.0/16" } ],
     "gateway": "10.10.10.1"
    }
   },
   {
    "type": "sbr"
   }
  ]
 }'
```

Pod creation goes smoothly, but once we delete the Pod, we get this CmdDel failure:
```
2021-04-14T00:39:02+02:00 [debug] Used defaults from parsed flat file config @ /etc/cni/net.d/whereabouts.d/whereabouts.conf
2021-04-14T00:39:02+02:00 [debug] DEBUG: &{Name: Type:whereabouts Routes:[] Datastore:kubernetes Addresses:[] OmitRanges:[] DNS:{Nameservers:[] Domain: Search:[] Options:[]} Range: RangeStart:<nil> RangeEnd:<nil> GatewayStr: EtcdHost: EtcdUsername: EtcdPassword: EtcdKeyFile: EtcdCertFile: EtcdCACertFile: LogFile:/var/log/whereabouts.log LogLevel:debug Gateway:<nil> Kubernetes:{KubeConfigPath:/etc/kubernetes/cluster-admin.kubeconfig K8sAPIRoot:} ConfigurationPath:}
2021-04-14T00:39:02+02:00 [error] IPAM configuration load failed: invalid CIDR : invalid CIDR address:
```
That extra DEBUG line is added by me to dump `n.IPAM` content. As we can see, most of the IPAM parameters are not set, especially the mandatory Range parameter.

**Comment by TothFerenc** [comment]
`host-local` IPAM behaves the same way. It cannot release the IP if the NAD has chained plugins inside.

Is this a Multus issue then?

**Comment by maiqueb** [comment]
Closing since the linked multus issue is marked as resolved, and offers a way for users to use CNI v0.4.0 and onwards.
